examples/products.js

```diff
@@ -0,0 +1,155 @@
+var fs = require('fs');
+const {dirname, join} = require('path');
+
+const leven = require('leven');
+
+const {dom, props, out, rule, ruleset, score, type} = require('../index');
+const {domSort, inlineTextLength, linkDensity, staticDom} = require('../utils');
+const {Annealer} = require('../optimizers');
+
+
+function euclideanDistance(nodeA, nodeB){
+  var rectA = nodeA.getBoundingClientRect();
+  var rectB = nodeB.getBoundingClientRect();
+  var x = rectB.left - rectA.left;
+  var y = rectB.bottom - rectB.bottom;
+  return Math.sqrt(x^2 + y^2);
+}
+
+function tunedImageFnodes() {
+    function imageSize(fnode) {
+      return fnode.element.offsetWidth * fnode.element.offsetHeight;
+    }
+
+    function imageHasSrc(fnode){
+      return fnode.element.hasAttribute('src') && fnode.element.getAttribute('src') != '';
```

Comment:
@erikrose Me thinks the ESLint rule you want is [`eqeqeq`](http://eslint.org/docs/rules/eqeqeq).